.promu.yml

```diff
@@ -0,0 +1,29 @@
+go:
+  version: 1.7.3
+repository:
+  path: github.com/jonnenauha/prometheus_varnish_exporter
+build:
+  flags: -a -tags netgo
+  ldflags: |
+    -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
+    -X {{repoPath}}/vendor/github.com/prometheus/common/version.Revision={{.Revision}}
+    -X {{repoPath}}/vendor/github.com/prometheus/common/version.Branch={{.Branch}}
+    -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
+    -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
```

Comment:
I do have most of these as well. You can get a version, a git hash and a build date by running with `-version`.